### PR TITLE
Mention how to enable serialization annotations

### PR DESCRIPTION
**Pull request by ogizanagi**
The bundle automatically enable the serializer, but not the annotations, despite the fact it's used in order to introduce serialization groups.
As the option itself isn't even documented yet on the Symfony Cookbook (~~I didn't search if you already have a pending PR~~ Of course you have :) ), I think it might be good to explain it in the bundle documentation.

### CODE REVIEW

File: Resources/doc/serialization-groups-and-relations.md [hunk 1]
```diff
@@ -21,6 +21,14 @@ services:
 The built-in controller and the Hydra documentation generator will leverage specified serialization and deserialization
 to give access only to exposed properties and to guess if they are readable or/and writable.
 
+## Annotations
+The Symfony serializer allows to specify the definition of serialization using XML, Yaml, or annotations. As annotations are really easy to understand, we'll use them in following examples.
```
Comment (dunglas) [inline Resources/doc/serialization-groups-and-relations.md]: Missing blank line.

Comment (dunglas) [inline Resources/doc/serialization-groups-and-relations.md]: YAML in camel case.

**Comment by dunglas** [comment]
Great! Can you update the TOC too in `README.md`?

**Comment by ogizanagi** [comment]
Done :)

**Comment by dunglas** [comment]
Thank you @ogizanagi

**Pull request merged**